comp/prizes/committee-award.yaml

```
@@ -7,4 +7,4 @@ component: Competition
 milestone: $SRYYYY Competition
 
 description: >-
-  The committee need to decide which team takes the committee award.
+  The competition team committee need to decide which team takes the committee award.
```

Is this strictly correct? I thought the conclusion we'd reached was that the committee that did the decided here was "the committee formed to decide the committee award"?

---

```suggestion
  Which team takes the committee award needs to be decided by the group tasked with doing so.
```

Avoiding the word "committee" as this becomes ambiguous with the team committees.

---

```suggestion
  The winner of the committee award needs deciding.
```